packages/medusa-plugin-brightpearl/src/services/brightpearl.js

```diff
@@ -695,22 +708,25 @@ class BrightpearlService extends BaseService {
         const bpProduct = await this.retrieveProductBySKU(variant.sku)
```

Comment:
suggestion: Is it possible to retrieve all the productBySKU first and then create a man and then looping throught the reservations?